model/riscv_insts_zicsr.sail

```diff
@@ -136,6 +148,13 @@ function readCSR csr : csreg -> xlenbits = {
     (0xB02,  _) => minstret[(sizeof(xlen) - 1) .. 0],
     (0xB80, 32) => mcycle[63 .. 32],
     (0xB82, 32) => minstret[63 .. 32],
+    (0x350,  _) => zero_extend(0b0),
+    (0x351,  _) => zero_extend(0b0),
```

Comment:
```suggestion
    (0x351,  _) => get_mireg(miselect[REG]),
```
This would allow extending it for future extensions by selecting on REG in the getter.